src/app/pages/ledger/connect-ledger-error/connect-ledger-error.layout.tsx

```diff
@@ -0,0 +1,75 @@
+import { FiCircle } from 'react-icons/fi';
+import { Box, color, Flex, Stack } from '@stacks/ui';
+
+import { BaseDrawer } from '@app/components/drawer';
+import { Caption } from '@app/components/typography';
+import ConnectLedgerError from '@assets/images/onboarding/ledger/connect-ledger-error.png';
+import { ErrorLabel } from '@app/components/error-label';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Link } from '@app/components/link';
+
+import { ConnectLedgerTitle } from '../components/connect-ledger-title';
+
+interface PossibleReasonUnableToConnectProps {
+  text: string;
+}
+function PossibleReasonUnableToConnect(props: PossibleReasonUnableToConnectProps) {
+  const { text } = props;
+
+  return (
+    <Flex>
+      <Box mr="tight" mt="3px">
+        <FiCircle fill={color('text-caption')} size="4px" />
+      </Box>
+      <Caption>{text}</Caption>
+    </Flex>
+  );
+}
+
+interface ConnectLedgerErrorLayoutProps {
+  onCancelConnectLedger(): void;
+  onTryAgain(): void;
+}
+export function ConnectLedgerErrorLayout(props: ConnectLedgerErrorLayoutProps) {
+  const { onCancelConnectLedger, onTryAgain } = props;
+
+  return (
+    <BaseDrawer title={<Box />} isShowing onClose={onCancelConnectLedger}>
+      <Stack alignItems="center" pb="loose" px="loose" spacing="loose" textAlign="center">
+        <Box mt="tight" width="247px">
+          <img src={ConnectLedgerError} />
+        </Box>
+        <ConnectLedgerTitle />
+        <ErrorLabel fontSize="14px" lineHeight={1.4}>
+          Unable to connect
+        </ErrorLabel>
+        <Stack
+          border="2px solid"
+          borderColor={color('border')}
+          borderRadius="12px"
+          spacing="base"
+          textAlign="left"
+          p="extra-loose"
+        >
+          <PossibleReasonUnableToConnect text="Check if Ledger Live is open. If it is, close it and try again" />
+          <PossibleReasonUnableToConnect text="Verify the Stacks app is installed" />
+          <PossibleReasonUnableToConnect text="If installed, verify you’re not on another app" />
+          <PossibleReasonUnableToConnect text="Verify the screen is unlocked" />
+          <PossibleReasonUnableToConnect text="Verify you chose the correct address to connect" />
+        </Stack>
+        <PrimaryButton height="40px" onClick={onTryAgain}>
+          Try again
+        </PrimaryButton>
+        {/* TODO: Add support page route link here */}
+        <Box>
+          <Caption>
+            If the problem persists, check our
+            <Link display="inline" onClick={() => {}}>
```

Comment:
Is there a space missing here?
<img width="121" alt="Screen Shot 2022-02-21 at 22 32 31" src="https://user-images.githubusercontent.com/1832037/155047114-975e6847-714c-46f4-919b-bd46da82124a.png">

Reply:
Thx, I'll fix that. The font size is too small too.

Reply:
Somewhat unrelated, but one rather annoying issue with `<Link />`, that causes us to sometimes write hard coded font sizes, is that itself has a default size. This is problematic, as it'd better default to _inheriting_ the container's font size.

Reply:
Couldn't agree more with @kyranjamie, a link component should not have a font size.